src/templates/developmentApi.tsx

```diff
@@ -40,8 +45,9 @@ const strFormat = str => {
 };
 
 export default props => {
-  const data = props.data?.openApi?.path || {};
-  const parameters =
+  const openApi: OpenAPI = props.data?.openApi;
```

Comment:
@NisanthanNanthakumar shouldn't the type here be `OpenAPI | undefined` since data might not have the `openApi` field?